### ISSUE TITLE
[Feature] Add --insert option (like in shell project)

### ISSUE THREAD

**Issue by ekalinin** [ekalinin/github-markdown-toc.go#26]
See:

- https://github.com/ekalinin/github-markdown-toc/issues/105

**Comment by damemi** [comment]
Hi! Have there been any updates on this? It would be very helpful for our project

**Comment by knennigtri** [comment]
Agree. I originally found the shell tool and loved the --insert feature... but then I wanted to use --depth, which I needed this project for. It would be ideal if both had both features, this is a great quick tool!
